functions/fisher_uninstall.fish

```diff
@@ -111,7 +111,7 @@ function fisher_uninstall -d "Disable / Uninstall Plugins"
             continue
         end
 
-        set -l tmp (mktemp)
+        set -l tmp (mktemp -t fisher.XXXXXXXX)
```

Comment:
On e.g. Ubuntu 14.04, the XXXXXX are required in a placeholder and get replaced with random characters (otherwise you get "mktemp: too few X's in template 'fisher'"). On FreeBSD and Mac OS X, they don't get replaced but they pose no problem.